src/main/java/org/apache/maven/plugins/scripting/EvalMojo.java

```diff
@@ -60,18 +64,27 @@ public class EvalMojo extends AbstractMojo {
     @Parameter
     String scriptResource;
 
+    @Component
+    private SettingsDecrypter settingsDecrypter;
+
     // script variables
     @Parameter(defaultValue = "${project}", readonly = true)
     private MavenProject project;
 
+    // script variables
+    @Parameter(defaultValue = "${session}", readonly = true)
```

Comment:
@Component can replace read-only parameter

Reply:
Is there a pro? I tend to prefer to limit the API (#imports) but if there is some advantage i'll do